CHANGELOG.md

```diff
@@ -11,6 +11,9 @@ Breaking changes
 * Client from @iov/bns was renamed to BnsConnection. The connect() function was renamed to BnsConnection.establish()
 * Address is now a string instead of an Uint8Array
 * In UserProfile, .entriesCount, .entryLabels and .entryIds have been merged into .wallets
+* @iov/core: rename `IovWriter` to `MultiChainSigner`
+* @iov/core: rename the getter `reader` to `connection` in MultiChainSigner
+* @iov-keycontrol: disallow the numeric indices in id
```

Comment:
last line is already added in #425. Please only add changelog entries for the current PR to make merging easier